### PR TITLE
added visualization updates for all assessments. Split out from PR #63

### PR DESCRIPTION
**Pull request by IanAtCredo**
## Change description

Splits out visualization updates from #63.

### CODE REVIEW

File: credoai/reporting/dataset_base.py [hunk 1]
```diff
@@ -47,134 +45,116 @@ def _plot_balance_metrics(self):
...
+                bbox_to_anchor=(1.2, 0.5), 
+                loc="center",
+                frameon=False,
+                ncol=num_classes,
+                title=label_name
+            )
+            ax.legend_.set_title(label_name)
+        self.figs.append(f)
 
     def _plot_group_diff(self):
```
Comment (amrasekh) [inline credoai/reporting/dataset_base.py:132]: This and the mutual information plots, which are both bar plots and would look better in landscape, are now plotted in portrait aspect ratio. 
image

File: credoai/reporting/plot_utils.py [hunk 2]
```diff
@@ -6,12 +6,62 @@
 
 DEFAULT_COLOR = '#4937c0'
 
-def get_style(rc=None, figsize=6, figure_ratio=1):
-    figsize = [figsize, figsize*figure_ratio]
-    style = {'figure.figsize': figsize,
-             'figure.dpi': 200}
```
Comment (amrasekh) [inline credoai/reporting/plot_utils.py:12]: For some reason, this `dpi` setting is not actually applied in some of the plots. If quickstart demo is run, this for example is some plots are made.
image
image

**Review by amrasekh — APPROVED** [review]
Included some suggestions/issues about the aspect ratio, text layout, and resolution of some of the plotting functions.

File: credoai/reporting/plot_utils.py (hunk 2 above)
Comment (IanAtCredo) [reply credoai/reporting/plot_utils.py:12]: I'll look into this.

File: credoai/reporting/dataset_base.py (hunk 1 above)
Comment (IanAtCredo) [reply credoai/reporting/dataset_base.py:132]: Good point - fixed

File: credoai/reporting/plot_utils.py (hunk 2 above)
Comment (IanAtCredo) [reply credoai/reporting/plot_utils.py:12]: Issue I found, which I have fixed.
* Jupyter defaults to displaying plots as png's.
*  It uses dpi as a scaling factor. So it isn't that easy to create a plot of a specific size and up the resolution.
*  I can make it so the plots are automatically created as svg's. This solves the issues (with some minor overhead for plotting). I've done this by adding the lines to all of our demo notebooks (and later to our reports).

**Pull request merged**